SwrveSDK/src/amazon/java/com/swrve/sdk/adm/SwrveAdmMessageReceiver.java

```diff
@@ -0,0 +1,10 @@
+package com.swrve.sdk.adm;
+
+import com.amazon.device.messaging.ADMMessageReceiver;
+
+//SwrveMessageReceiver listens for messages from ADM
```

Comment:
Yes, we define (or ask customers to define) the receiver in their manifest.
 @Sergio-Mira requested it be moved from inner static class in earlier comment/review.